### PR TITLE
fix: auth screen

### PR DESCRIPTION
**Pull request by Coder-Manan**

### CODE REVIEW

File: lib/presentation/login/components/oauth_webview.dart [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: ValueListenableBuilder(
+          valueListenable: _loadingState,
+          builder: (context, value, _) {
+            return IndexedStack(
+              index: int.parse(value.toString()),
```
Comment (aman-singh7) [inline lib/presentation/login/components/oauth_webview.dart:23]: why aren't you passing the value directly?

File: lib/presentation/login/components/oauth_webview.dart [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+import 'package:auto_route/auto_route.dart';
+
+@RoutePage()
+class OAuthWebScreen extends StatelessWidget {
+  static const id = 'oauth_view';
+  OAuthWebScreen({Key? key}) : super(key: key);
+  final ValueNotifier<int> _loadingState = ValueNotifier(1);
+  final String omniportSignUpURL =
+      'https://channeli.in/oauth/authorise/?client_id=${EnvironmentConfig.OAUTH_CLIENT_ID}&redirect_uri=${EnvironmentConfig.OAUTH_REDIRECT_URI}';
```
Comment (aman-singh7) [inline lib/presentation/login/components/oauth_webview.dart:13]: add this to AppStrings

**Review by aman-singh7 — CHANGES_REQUESTED** [review]

**Comment by aman-singh7** [comment]
@Coder-Manan fix the ci

File: lib/presentation/login/components/oauth_webview.dart (hunk 1 above)
Comment (Coder-Manan) [reply lib/presentation/login/components/oauth_webview.dart:23]: Fixed

File: lib/presentation/login/components/oauth_webview.dart (hunk 2 above)
Comment (Coder-Manan) [reply lib/presentation/login/components/oauth_webview.dart:13]: Done

**Comment by aman-singh7** [comment]
@Coder-Manan fix the ci and resolve the conflicts.

**Review by aman-singh7 — APPROVED** [review]

**Pull request merged**